.pre-commit-config.yaml

```diff
@@ -28,7 +28,10 @@ repos:
         verbose: true
         args: [--soft-fail]
 
-  - repo: https://github.com/Wynndow/spotless-pre-commit.git
-    rev: 'v0.2.0'
+  - repo: local
     hooks:
       - id: spotlessCheck
+        name: Run spotless check
+        entry: ./gradlew spotlessCheck
+        language: system
+        pass_filenames: false
```

Comment:
For bonus points, can we restrict this to only run if you modify a java file?

Reply:
Every Little Helps